wren-ui/src/apollo/server/resolvers/modelResolver.ts

```diff
@@ -329,7 +329,7 @@ export class ModelResolver {
 
     // if description is not null, or undefined, update the description in properties
     if (!isNil(data.description)) {
-      const properties = JSON.parse(model.properties);
+      const properties = JSON.parse(model.properties) || {};
```

Comment:
@fredalai what is the value of `model.properties` that causes this issue ?

Reply:
I see, maybe we can change to sth like `isNil(model.properties) ? {} : JSON.parse(model.properties)`

If properties is null or undefined, we don't have to JSON parse it.